src/controllers/update/check.your.answers.controller.ts

```diff
@@ -33,9 +38,17 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
     logger.debugRequest(req, `${req.method} ${req.route.path}`);
 
     const session = req.session as Session;
+    const appData: ApplicationData = getApplicationData(session);
 
-    const transactionID = await postTransaction(req, session);
-    const overseasEntityID = await createOverseasEntity(req, session, transactionID);
+    let transactionID, overseasEntityID;
+    if (isActiveFeature(FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022)) {
```

Comment:
This is about save & resume and not the skeletal filing page.

Reply:
This is the first page where SnR "save" gets executed hence the implementation is done as part of this JIRA.

Reply:
Would have been cleaner to add page first then save&resume but as we are here.